lib/validators/properties-validator.ts

```diff
@@ -13,6 +13,18 @@ import * as semver from 'semver';
 
 const RESERVED = [/^parallax$/];
 
+const TYPES_ALLOWING_CHILD_PROPERTIES = [
```

Comment:
Wonder if you could match the type to `ComponentPropertyControl['type'][]`? Would prevent typos.

https://github.com/WoodWing/studio-component-set-tools/blob/master/lib/models/component-property-controls.ts#L3

Reply:
Good point